src/app/pages/feed/feed.component.html

```diff
@@ -53,8 +53,8 @@ <h1 class="title">Your activities</h1>
   </div>
 </div>
 <div class="addNew">
-  <button class="newActivity">
-    Add new
+  <button class="newActivity" (click)="showNewActivity()">
```

Comment:
```suggestion
  <button class="newActivity" (click)="showNewActivityModal()">
```
This is opening a modal, right?